### ISSUE TITLE
[New Language Support]: Swedish/Svenska

### ISSUE THREAD

**Issue by qwerty13547** [PRProd/HA-Firemote#472]
### Language Code

SV

### Translation Chart

```text
01. App Launch Button => Appstartknapp
02. App Launcher Relative Scale => Appstartknapparnas Storlek
03. Apple Computers => Apple Datorer
04. Apple Music => Apple Musik
05. Apple Photos => Apple Foton
06. Associated => Associerad
07. bottom => botten
08. Compatibility Mode => Kompabilitetsläge
09. Computers => Datorers
10. Custom Remote Skin => Anpassat Skal För Fjärrkontroll
11. Default for => Standard För
12. Device Family => Enhetsfamilj
13. Device Model => Enhetsmodell
14. Device Name Text Color => Enhetsnamn
15. DPad Style => DPad Stil
16. Entity => Entitet
17. Function: Captions => Funktion: Textning
18. Function: Channel + => Funktion: Kanal +
19. Function: Channel - => Funktion: Kanal -
20. Function: Control Center => Funktion: Kontrollcenter
21. Function: Find My Remote => Funktion: Hitta Min Fjärrkontroll
22. Function: Mute => Funktion: Tysta
23. Function: Next => Funktion: Nästa
24. Function: Pairing => Funktion: Para
25. Function: Previous => Funktion: Föregående
26. Function: Reboot => Funktion: Starta om
27. Function: Search => Funktion: Sök
28. Function: Settings => Funktion: Inställningar
29. Function: Skip Backward => Funktion: Hoppa Bakåt
30. Function: Skip Forward => Funktion: Hoppas Framåt
31. Function: Switch Apps => Funktion: Växla Appar
32. Function: System Update => Funktion: Uppdatera Systemet
33. Function: Volume + => Funktion: Volym +
34. Function: Volume - => Funktion: Volym -
35. hidden => dold
36. Hide frames around button groups => Dold Ram Runt Knappgrupper
37. Music => Musik
38. Name Position => Placering Av Namn
39. optional => valfri
40. Photos => Foton
41. Remote Style => Fjärrkontrollsstil 
42. Scale => Storlek
43. Settings => Inställningar
44. Search => Sök
45. top => topp
46. Visible Device Name => Synligt Enhetsnamn
```

### Additional Comments (optional)

Due to grammar there are some I need to put into context to know how it should be bent. (6,7,9,31,35,38,39,42)

**Comment by PRProd** [comment]
Thanks for your help with this @qwerty13547!

> Due to grammar there are some I need to put into context to know how it should be bent. (6,7,9,31,35,38,39,42)

Absolutely!  We can work together in this thread, and eventually I will release a beta version for you to test.  You are the expert here, so ultimately you have the final decision in what is best for this language translation.  If you find anything wrong, big or small, let me know and I'll be happy to adjust it.

### Firemote Translation Process
 1. The first thing I do when I receive a new language request is to format the data in a way the code requires.  Everything shown below will be in that new format.
 2. If the **Apple TV source_list** field is empty, I remove the translations that were provided that apply only to AppleTV devices.  Because of this step, you will see fewer translated items than you originally provided.
 3. I review the translations with online translator tools to see if anything looks like it might need attention.  I'll sometimes adjust things to what the tools suggest is a better translation, but ultimately I will let the requester (you) make the final decision.
 4. I make sure the translations are applying as expected, then provide the requester (you) with usage context.  Usually this is in the form of screenshots.
<br>

This is the outcome of steps 1-3:
![image](https://github.com/user-attachments/assets/a3c279b0-b8b9-4d6b-9e2e-8ce11edc061d)

Here are some (step 4) examples of the applied translations:
![image](https://github.com/user-attachments/assets/9dfb0763-e9a2-43bc-b2e0-5d191045f000)

![image](https://github.com/user-attachments/assets/117df371-2eaf-40a6-aea2-dd905d1a4d9c)

![image](https://github.com/user-attachments/assets/50ff2176-ab41-422b-9293-0e2180e93524)

![image](https://github.com/user-attachments/assets/e4814911-0f92-43ef-b494-499bb480a19d)

![image](https://github.com/user-attachments/assets/eaf7260f-820f-4409-ad73-dffda477c92c)


### Next Steps
Now it's your turn!  Take a look at the images that I've provided and let me know what you think.  I understand that the pictures I've provided do not represent 100% of the translations you provided, so if there is something else that you want to see please let me know.  I'm also aware that I changed a couple of things, so I want your opinions about that too.

Thanks for your help!

-Doug

**Comment by PRProd** [comment]
@qwerty13547 ~

I'm hoping you've had a chance to review my previous post.  I have just published a beta Firemote version [v4.0.0b1](https://github.com/PRProd/HA-Firemote/releases/tag/v4.0.0b1) which contains this translation.  Feel free to download the beta version to try it out, or choose to wait until the next official Firemote release.  I will leave this issue open until you are satisfied with the results, and they have been pushed to an official release.  Enjoy!

-Doug
<hr>

After updating to the newest version, if you are unable to see the new updates on your Firemote card, visit [this link](https://github.com/PRProd/HA-Firemote/wiki/Force-a-Refresh) for help.
After updating to the newest version, if you are unable to see language updates, visit [this link](https://github.com/PRProd/HA-Firemote/wiki/Changing-Your-Home-Assistant-Language) for help.

**Comment by qwerty13547** [comment]
Hi Doug,

Sorry for not replying sooner. I have a new kid and he takes alot of my time for the moment.
I have in stalled the beta and looked around. 

I have found a few gramatical things. I attached them as screenshots below.
Let me know it I have missed anything.  

![Caption](https://github.com/user-attachments/assets/b0198b5c-3d09-4962-9210-17a184e8fe9a)
![compatibility mode](https://github.com/user-attachments/assets/304caab3-4875-4888-860d-3111ed9fc7ac)
![Computers](https://github.com/user-attachments/assets/f3874221-990b-4839-b5b6-e6d0a5f533c0)
![Device family](https://github.com/user-attachments/assets/68324166-59c2-4fd5-a673-ee2301ea061e)
![Device name text color](https://github.com/user-attachments/assets/86e87b9d-668e-4b7d-9514-9232813b8fbc)
![Name position](https://github.com/user-attachments/assets/634dd3ce-6c0b-4377-8fb2-a58368a2f212)
![Ramar runt](https://github.com/user-attachments/assets/da9d70f3-e067-4b69-a7cc-4c1c11ac3c48)
![Remote style](https://github.com/user-attachments/assets/25bc5bcc-5253-454a-90ac-2717784bad4a)

**Comment by PRProd** [comment]
@qwerty13547 ~

Thank you for your thorough reply!  🎉 Congrats on your new kid too! 👶🏻 

Here are some screenshots showing the corrections that you have made.  I will be releasing these in the next beta for you to look at.  Meanwhile, if you see anything wrong here, or you find anything else that should be adjusted, please let me know.  THANK YOU for your help!

![image](https://github.com/user-attachments/assets/4a5f2707-94eb-4e2f-b291-4ec3df2451e4)

![image](https://github.com/user-attachments/assets/e4fe2b31-9694-4f12-b225-d6b23622f173)
I'm opting to leave the word "event" untranslated because in this context, agnostic of language, the Android/Linux file system always refers to these input channels as event0, event1, etc.
<br>
**Computers** was [removed from this translation map](https://github.com/PRProd/HA-Firemote/issues/472#issuecomment-2269579200) because it is an Apple TV app name, not a word that appears in the configuration UI. (more information about that can be found [here](https://github.com/PRProd/HA-Firemote/wiki/#which-languages-are-currently-supported))
<br>

![image](https://github.com/user-attachments/assets/e76cf783-8d95-4050-9bc0-b076b3c566f8)

![image](https://github.com/user-attachments/assets/06c23e06-092a-4771-88ed-25d94b676a12)

![image](https://github.com/user-attachments/assets/6e0df3dd-a81c-4b39-b426-2a5372df652a)

![image](https://github.com/user-attachments/assets/f8c890a8-d017-4eed-80ef-199eb47812f5)

![image](https://github.com/user-attachments/assets/6d773ca8-3ae3-4dd8-81b0-b38126626781)

### New Stuff
I also have a couple more items for your review:
![image](https://github.com/user-attachments/assets/467cf810-6143-49bd-8501-eeceae55ecd5)
black = svart
silver = silver (no change on this one)
Minimal = Minimal (no change on this one)

**Comment by qwerty13547** [comment]
Thank you! It is fun but takes alot of time and energy 🥱
I have no objection. All those screenshot are correct  👍

I will do another check efter next beta

**Comment by PRProd** [comment]
I have just published Firemote beta version [v4.0.0b2](https://github.com/PRProd/HA-Firemote/releases/tag/v4.0.0b2).  Let me know if you find anything else!  😄

**Comment by qwerty13547** [comment]
Now, I can not find anything to address ✨

**Comment by PRProd** [comment]
An update for you @qwerty13547 ~

I have just published Firemote version [v4.0.0](https://github.com/PRProd/HA-Firemote/releases/tag/v4.0.0) which moves this language support request from beta to official.  Enjoy!

-Doug
<hr>

After updating to the newest version, if you are unable to see the new updates on your Firemote card, visit [this link](https://github.com/PRProd/HA-Firemote/wiki/Force-a-Refresh) for help.
After updating to the newest version, if you are unable to see language updates, visit [this link](https://github.com/PRProd/HA-Firemote/wiki/Changing-Your-Home-Assistant-Language) for help.
